### PR TITLE
webrtc: allow configuring timeouts (#3404)

### PR DESCRIPTION
**Pull request by jwalton**
Allow configuration of WebRTC timeouts.

### CODE REVIEW

File: internal/core/core.go
Comment (aler9) [inline internal/core/core.go:1]: in line 853, add a comparison between current and previos WebRTCHandshakeTimeout p.conf.WebRTCTrackGatherTimeout in order to detect changes after a API request or hot reloading

**Review by aler9 — CHANGES_REQUESTED** [review]
thanks for this patch, take a look at the comment in the review

File: internal/core/core.go
Comment (jwalton) [reply internal/core/core.go:1]: Fixed.

**Review by aler9 — APPROVED** [review]

**Comment by aler9** [comment]
merged, thanks!

**Pull request merged**